src/style.css

```diff
@@ -1 +1,26 @@
 /* add styles here */
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star--active {
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+}
```

Comment:
Remember to use fallback fonts - although not directly related to the lines you've modified, it's a good practice to include fallback fonts in your CSS in case the main font doesn't load. Consider adding a generic font family as a fallback to your `body` or other text-containing elements.